build/site/404.html

```diff
@@ -69,9 +69,6 @@
         <li class="version is-latest">
           <a href="/specifications/specifications/1.3/index.html">1.3</a>
         </li>
-        <li class="version">
```

Comment:
This looks like removing references to 1.2. Intended or am I misunderstanding? A little further down, too.